src/test/java/seedu/address/testutil/LeaveListBuilder.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.testutil;
+
+import seedu.address.model.LeaveList;
+import seedu.address.model.leave.Leave;
+
+//@@author Hafizuddin-NUS
+/**
+ * A utility class to help with building LeaveList objects.
+ * Example usage: <br>
+ *     {@code LeaveList ll = new LeaveListBuilder().withLeave("REQUEST_1", "REQUEST_1").build();}
```

Comment:
Can backspace the extra spaces here

Reply:
Need space there. Same goes to AddressBookBuilder